microservices-connector/cmd/router/main.go

```diff
@@ -91,8 +96,15 @@ func callService(step *gmcv1alpha3.Step, serviceUrl string, input []byte, header
 	defer timeTrack(time.Now(), "step", serviceUrl)
 	log.Info("Entering callService", "url", serviceUrl)
 
+	// log the http header from the original request
+	log.Info("Print the http request headers", "HTTP_Header", headers)
```

Comment:
The info message does not need to contain "Print the http request headers", just "HTTP_request_header" enough

Reply:
Hi @mkbhanda, this operation is just keep the `header` of HTTP request which should be used for handling some feature later.